### ISSUE TITLE
Constant syncing issue 

### ISSUE THREAD

**Issue by churik** [status-im/status-react#11351]
# Bug Report

## Problem
Sometimes Status app shows constant syncing bar, however internet connection is stable.
Reported by @jarradh, creating an issue to keep track on it.

#### Expected behavior
Status connected, no connectivity bar is shown

#### Actual behavior
> Status continually syncing 

### Reproduction

[comment]: # (Describe how we can replicate the bug step by step.)

> - Open Status
> - Watch Status sync for longer than 20 seconds sometimes new messages come through but still acts as if syncing 
> - Status continually syncing 

### Additional Information
----------------------------------------
App version: 1.8.0 (20201019145001)
OS: android
Node version: StatusIM/v0.62.11/android-arm64/go1.14.7
Mailserver: mail-02.ac-cn-hongkong-c.eth.prod

#### Logs

28/10/20:
[Status-debug-logs.zip](https://github.com/status-im/status-react/files/5473701/Status-debug-logs.zip)

02/11/20:
[Status-debug-logs.zip](https://github.com/status-im/status-react/files/5473699/Status-debug-logs.zip)

**Comment by churik** [comment]
cc @cammellos

**Comment by cammellos** [comment]
Investigating this.
Replicated the behavior by hitting a request with the same number of topics to the mailserver and the request expires.
So this is clearly due to the number of topics. Will investigate further.

**Comment by cammellos** [comment]
From Europe hitting amsterdam does not timeout:

```
enode://7aa648d6e855950b2e3d3bf220c496e0cae4adfddef3e1e6062e6b177aec93bc6cdcf1282cb40d1656932ebfdd565729da440368d7c4da7dbd4d004b1ac02bf8@178.128.142.26:443
```

while hitting HK:

```
enode://e85f1d4209f2f99da801af18db8716e584a28ad0bdc47fbdcd8f26af74dbd97fc279144680553ec7cd9092afe683ddea1e0f9fc571ebcb4b1d857c03a088853d@47.244.129.82:443 
```

Does timeout. 

I will check whether is dependent on the data  exchanged, as it might not be (publishing might be asynchronous, but will double check)

**Comment by cammellos** [comment]
Most of the times it actually works fine with HK server as well.
One thing I did notice though is that the bloom filter for that query is essentially full, which means that any envelope will be sent back, likely the source of the issue.
Interesting is that my understanding is that we should not be using bloomfilters for mailserver queries anymore after waku, or so I was told, but it seems like it might still be the case.

**Comment by cammellos** [comment]
Confirmed that is definitely not using topics but bloom filter :( 
good news is that probably the change will have a significant impact on bw usage, but it is  a change nonetheless

**Comment by cammellos** [comment]
Will be tracked separately https://github.com/status-im/status-go/issues/2068

**Comment by cammellos** [comment]
Closing this for now, as the most likely culprit was using bloom filters instead of topics in the query. I have tested it in an artificial environment and it should solve the issue, but the ultimate test is the users. 
Will be released in 1.10. Feel free to re-open if it's still an issue in 1.10.
